asyncvnc.py

```diff
@@ -285,14 +285,28 @@ class Video:
     data: Optional[np.ndarray] = None
 
     @classmethod
-    async def create(cls, reader: StreamReader, writer: StreamWriter) -> 'Video':
+    async def create(
+        cls,
+        reader: StreamReader,
+        writer: StreamWriter,
+        force_mode: Optional[str] = None,
+
+    ) -> 'Video':
         writer.write(b'\x01')
         width = await read_int(reader, 2)
         height = await read_int(reader, 2)
-        mode_data = bytearray(await reader.readexactly(13))
+        mode_bits = await reader.readexactly(13)
+        mode_data = bytearray(mode_bits)
         mode_data[2] &= 1  # set big endian flag to 0 or 1
         mode_data[3] &= 1  # set true colour flag to 0 or 1
         mode = video_modes.get(bytes(mode_data))
+
+        # # allow user to force a ``mode: str`` if they know it works.
```

Comment:
will drop the history that brought in this patch set if you're happy with the test suite @barneygale.

So in other words I'll drop the history from #3 before you merge this.